### PR TITLE
[EduPage] Added ruleset

### PR DESCRIPTION
**Pull request by Roman3349**
Added rule for school management system EduPage.

### CODE REVIEW

**Comment by fuglede** [comment]
Thanks for the ruleset; unfortunately, this breaks http://mobile.edupage.org/. On the other hand, https://classbook.edupage.org/ seems to work fine. The notes on using wildcards in rules from [the ruleset style guide](https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md) are kind of relevant here.

**Comment by Roman3349** [comment]
I use wildcard in this ruleset because most subdomains run on HTTPS correctly (for example:
* http://gymbos.edupage.org/,
* http://zsgogolovato.edupage.org/,
* http://sp2miasteczko.edupage.org/,
* http://zsltepla.edupage.org/,
* http://msp11piekary.edupage.org/,
* http://gymsu.edupage.org/,
* http://zskomjatice.edupage.org/,
* etc.).

**Comment by Roman3349** [comment]
Now it should be fine.

File: src/chrome/content/rules/EduPage.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<ruleset name="EduPage">
+    <target host="edupage.org" />
+    <target host="*.edupage.org" />
+
+    <exclusion pattern="^http://mobile\.edupage\.org/"/>
```
Comment (loveithateit) [inline src/chrome/content/rules/EduPage.xml]: help.edupage.org isn't served over https either.

**Comment by Roman3349** [comment]
@loveithateit Ruleset repaired. Tests failed because hostname (www.)danskebank.dk shows up in 2 different rulesets.

**Pull request merged**